frontend/src/pages/projects/projectSharing/data.ts

```diff
@@ -0,0 +1,44 @@
+import { RoleBindingKind } from '~/k8sTypes';
+import { SortableData } from '~/utilities/useTableColumnSort';
+
+export const columnsProjectSharingUser: SortableData<RoleBindingKind>[] = [
+  {
+    field: 'username',
+    label: 'Username',
+    width: 30,
+    sortable: (a, b) => a.roleRef.name.localeCompare(b.roleRef.name),
+  },
+  {
+    field: 'permission',
+    label: 'Permission',
+    width: 20,
+    sortable: false,
```

Comment:
both permission and date fields are not sortable, not sure if we want that @kywalker-rh

Reply:
Wow, maybe i mixed other feature, that's right, ther is no sorting optiom, if that's so ill remove it cc @kywalker-rh

Reply:
I think sorting is a good idea, I just missed adding it to the mocks. If its easy to add I'd put sorting in for all 3 fields. From a usability standpoint, it may not be used, but if you want it and its helpful its a great feature to have.

Reply:
Great, is quite easy to implement 👍